admin/src/app/users/users.component.html

```diff
@@ -104,10 +104,10 @@ <h2 class="users-header">
           <th mat-header-cell *matHeaderCellDef>Actions</th>
           <td mat-cell *matCellDef="let user">
             <button mat-icon-button color="accent" (click)="editUser(user.id)">
-              <mat-icon>edit</mat-icon>
+              <img src="assets/svg/edit.svg" alt="edit">
```

Comment:
use [svg](https://www.npmjs.com/package/angular-svg-icon) for showing svgs, as implemented in practitioner